boxoffice/views/admin_order.py

```diff
@@ -123,3 +123,53 @@ def admin_org_order(org, order):
     line_items = LineItem.query.filter(LineItem.order == order,
         LineItem.status.in_([LINE_ITEM_STATUS.CONFIRMED, LINE_ITEM_STATUS.CANCELLED])).all()
     return dict(org=org, order=order, line_items=line_items)
+
+
+@app.route('/api/1/organization/<org_name>/order/<int:receipt_no>', methods=['GET'])
+@load_models(
+    (Organization, {'name': 'org_name'}, 'org'),
+    (Order, {'organization': 'org', 'receipt_no': 'receipt_no'}, 'order')
+)
+def order_api(org, order):
+    check_api_access(org.details.get('access_token'))
+
+    line_items_list = [{
+    'title': li.item.title,
+    'status': LINE_ITEM_STATUS[li.status],
+    'base_amount': li.base_amount,
+    'discounted_amount': li.discounted_amount,
+    'final_amount': li.final_amount
+    } for li in order.confirmed_cancelled_line_items]
+
+    invoices_list = [{
+        'status': INVOICE_STATUS[invoice.status],
+        'invoicee_company': invoice.invoicee_company,
+        'invoicee_email': invoice.invoicee_email,
+        'invoice_no': invoice.invoice_no,
+        'invoiced_at': invoice.invoiced_at,
+        'street_address_1': invoice.street_address_1,
+        'street_address_2': invoice.street_address_2,
+        'city': invoice.city,
+        'state': invoice.state,
+        'state_code': invoice.state_code,
+        'country_code': invoice.country_code,
+        'postcode': invoice.postcode,
+        'buyer_taxid': invoice.buyer_taxid,
+        'seller_taxid': invoice.seller_taxid,
+    } for invoice in order.invoices]
```

Comment:
As per our style guide, this last line should be indented as well. This is a departure from PEP8, which demands this line not be indented.